src/app/game-list/game-list.component.ts

```diff
@@ -0,0 +1,54 @@
+import { Component, OnInit, Input } from '@angular/core';
+import { Game } from '../types/game/Game';
```

Comment:
Nice job adding types! Let's add all types to the `types/index.ts` file so that external imports can be from just `types/`